src/components/ProfileDetails.js

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import { useParams } from "react-router-dom";
+import userProfiles from "../profileData";
+export default function ProfileDetails() {
+  const { username } = useParams();
+  const currentUser = userProfiles.find((user) => +user.id === +username);
```

Comment:
I think I need to change +username to +userid or +id, becuase currentUser is found by user.id